lib/websocket.js

```diff
@@ -21,10 +21,13 @@ export class WebsocketClient extends BaseClient {
 	 * Start the websocket client.
 	 */
 	start() {
-		this.client = new websocket.client();
-		this.client.on('connectFailed', () => {});
+		this.client = new websocket.client({ rejectUnauthorized: false });
+		this.client.on('connectFailed', (e) => {/*console.log(`Websocket connection error: ${e.message}`)*/});
```

Comment:
There is no mechanism for establishing a connection yet, at least you can show a `console.error()` so that the operator knows what is going on?

Reply:
changed it to console.log and uncommented